src/RegisterConfig.ts

```diff
@@ -0,0 +1,96 @@
+import { Registers as reg } from "./Registers"
+import { type Config } from "./OS8104A"
+
+export function getRegisterConfig(
+    { nodeAddressLow, nodeAddressHigh, groupAddress }: Config,
+    mode: number
+): Map<number, number> {
```

Comment:
Can be `Map<reg, number>`

Reply:
https://github.com/rhysmorgan134/SocketMost/blob/f37c500d3a2438e7080c5f92f6c5ae798d93f5d0/src/driver/RegisterConfig.ts#L4-L7